### PR TITLE
[#169206663] Fixed Markdown ActivityIndicator visibility

### PR DESCRIPTION
**Pull request by Alex193a**
The space occupied by the _ActivityIndicator_, in the Markdown component, is not removed when it is not animating.

**BEFORE:**

**AFTER:**

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🐞 [#169206663](https://www.pivotaltracker.com/story/show/169206663): Nel componente Markdown, non viene rimosso lo spazio occupato dall’ActivityIndicator quando questo non è animato

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/components/ui/Markdown/index.tsx [hunk 1]
```diff
@@ -231,15 +231,18 @@ class Markdown extends React.PureComponent<Props, State> {
       height: htmlBodyHeight
     };
 
+    const isLoadingComplete =
```
Comment (Undermaken) [inline ts/components/ui/Markdown/index.tsx]:
```suggestion
    const isLoading =
```
Is it a better naming?

Comment (Alex193a) [reply ts/components/ui/Markdown/index.tsx]: Yup, changed.

File: ts/components/ui/Markdown/index.tsx [hunk 2]
```diff
@@ -231,15 +231,18 @@ class Markdown extends React.PureComponent<Props, State> {
...
-          color={customVariables.brandPrimary}
-          animating={
-            html === undefined || (html !== "" && htmlBodyHeight === 0)
-          }
-        />
+        {isLoading && (
+          <ActivityIndicator
+            size="large"
+            color={customVariables.brandPrimary}
+            animating={isLoading}
```
Comment (Undermaken) [inline ts/components/ui/Markdown/index.tsx]:
```suggestion
            animating={true}
```

**Pull request merged**